clusterloader2/cmd/clusterloader.go

```diff
@@ -269,11 +269,15 @@ func main() {
 		klog.Errorf("http server unexpectedly ended: %v", err)
 	}()
 
-	provider, err := provider.NewProvider(&providerInitOptions)
+	newProvider, err := provider.NewProvider(&providerInitOptions)
 	if err != nil {
 		klog.Exitf("Error init provider: %v", err)
 	}
-	clusterLoaderConfig.ClusterConfig.Provider = provider
+	clusterLoaderConfig.ClusterConfig.Provider = newProvider
```

Comment:
This is not ideal that we have it hardcoded here, but there is not good place for it, I will follow-up with moving that flag to provider-specific option instead.